synfig-studio/src/gui/canvasview.cpp

```diff
@@ -2728,6 +2746,17 @@ CanvasView::toggle_onion_skin()
 	toggling_onion_skin=false;
 }
 
+void
+CanvasView::toggle_onion_skin_keyframes()
+{
+	if(toggling_onion_skin_keyframes)
```

Comment:
Due to the current implementation strategy, you have to change `CanvasView::on_meta_data_changed()` too.

Reply:
As of now it's not saving any metadata, is it better to do so?